datamap/examples/src/minmaxdatamap/main/java/org/apache/carbondata/datamap/examples/MinMaxIndexDataMapFactory.java

```diff
@@ -137,7 +137,7 @@ public DataMapBuilder createBuilder(Segment segment, String shardName,
    * @param segment
    */
   @Override
-  public void clear(Segment segment) {
+  public void clear(String segmentNo) {
```

Comment:
These changes are not related to this  issue, just fix  some compile errors after PR#3434.